deploy/helm/magda/charts/opa/policies/object/dataset/allow_test.rego

```diff
@@ -0,0 +1,543 @@
+package object.dataset
+
+test_allow_no_constraints_permission_user_without_no_constraints_permission {
+    not allow with input as {
+        "operationUri": "object/dataset/draft/read",
+        "object": {
+            "dataset": {
+                "accessControl": {
+                    "ownerId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx",
+                    "orgUnitOwnerId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx",
+                    "preAuthoisedPermissionIds": ["xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"]
```

Comment:
Authorised?

Reply:
You found it~ 😭 I just found it last week and it's corrected on issue/2154 with this commit:
https://github.com/magda-io/magda/pull/2175/commits/609798d0ce56d2e88d19415b557cf750d425d855